### PR TITLE
Only show non-guest users in the sidebar

### PR DESCRIPTION
**Pull request by GGH-WeyboZetO**
### Component/Part
Frontend

### Description
Only show non-guest users in the sidebar

### Steps
- [x] Added implementation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md) and
  made sure that:
  - My commits are signed-off to accept the DCO.
  - This PR targets the correct branch: `develop` for 2.x
  - Signed-off-by: Shan Michel <yuplasmas@gmail.com>

### Related Issue(s)
#2953

### CODE REVIEW

**Comment by mrdrogdrog** [comment]
Hello,

thanks for your contribution, but I guess you misunderstood something here. With "sidebar" @ErikMichelson meant the sidebar in the revisions dialog. Or in other words: Don't list the guest users in the list items.

While this would be the easiest solution, it isn't the preferred one. The alternative in an issue should only be implemented if the requested feature/improvement is not possible.

**Comment by GGH-WeyboZetO** [comment]
Hello,

I think I understand now what he meant now.
REvision
I'm doing my tests with yarn start:dev with localhost but I don't see any "mock icon" like that:
hw
How can I get something similar with 1-2 mock icons or have that feature not been deleted?
The issue is to get the name of the user displayed when we hover our mouse over the icon right?

Thank you.

**Comment by mrdrogdrog** [comment]
> I'm doing my tests with yarn start:dev with localhost but I don't see any "mock icon" like that:

that's because the backend doesn't save author information right now. It simply just doesn't know who edited what. 

>  or have that feature not been deleted

You can't. if you run the frontend in mock mode then there is NO data persistence. If you need other data then you have to change the mock data in `src/pages/api/private/notes/feature/revisions/`.


> The issue is to get the name of the user displayed when we hover our mouse over the icon right

No. That is already working. The original issue #2953 is about showing how many guest users have participated in a revision.

**Comment by GGH-WeyboZetO** [comment]
I tried to git grep "revisions" and got that result but since it's a frontend issue I guess none of them are right since they are all backend.
grepdd
Can I ask where exactly is located the done doing thoses boxes?
lcasx

**Comment by GGH-WeyboZetO** [comment]
modifnombreauteur

File: frontend/src/components/editor-page/document-bar/revisions/revision-list-entry.tsx [hunk 1]
```diff
@@ -74,6 +75,10 @@ export const RevisionListEntry: React.FC<RevisionListEntryProps> = ({ active, on
         </ShowIf>
         <ShowIf condition={!revisionAuthors.error && !revisionAuthors.loading}>{revisionAuthors.value}</ShowIf>
       </span>
+      <span>
+        <UiIcon icon={IconPersonPlus} className='mx-2' />
+        <Trans i18nKey={'Number of author'} />: {revision.anonymousAuthorCount}
```
Comment (mrdrogdrog) [inline frontend/src/components/editor-page/document-bar/revisions/revision-list-entry.tsx:80]: We don't use actual sentences as i18n keys but fqdn like in line 69.

**Review by mrdrogdrog — CHANGES_REQUESTED** [review]
Please squash your commits.

**Pull request closed without merging**